src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -47,7 +48,11 @@ public function get_checkout_template_vars() {
 	 * @return bool
 	 */
 	public function include_payment_element() {
-		return tribe_get_option( Settings::$option_checkout_element ) === Settings::PAYMENT_ELEMENT_SLUG;
+		if ( tribe_get_option( Settings::$option_checkout_element ) !== Settings::PAYMENT_ELEMENT_SLUG  ) {
+			return false;
+		}
+		$payment_methods = ( new Merchant() )->get_payment_method_types();
```

Comment:
A couple line breaks for readability
```suggestion

		$payment_methods = ( new Merchant() )->get_payment_method_types();

```